js/index.js

```diff
@@ -58,45 +68,48 @@ const makeRequest = fullURL => {
       responseContent = httpRequest.responseText;
       parsed = JSON.parse(responseContent);
       displayContent(parsed);
-      console.log(parsed);
     }
   };
 
   httpRequest.open('GET', fullURL);
   httpRequest.send();
 
-  const displayContent = parsed => {
+  const displayContent = ({ data }) => {
+    //used destructuring for parsed.data => parsed object is destructured and {data} takes it's value.
     let id = 0;
     htmlContainerElement.innerHTML = '';
-    for (let key in parsed.data) {
+    for (let key in data) {
       const img = document.createElement('img');
-      const imgSrc = parsed.data[key].images.fixed_width.url;
-      img.style.margin = '0.5rem 0.5rem';
-      img.src = imgSrc;
-      img.id = id;
-      img.setAttribute('onclick', 'openModal()');
+      //used destructuring
+      const [
+        margin,
+        {
+          images: {
+            fixed_width: { url },
+          },
+        },
+      ] = ['0.5rem 0.5rem', data[key]];
```

Comment:
I'd say that the logic of destructuring here is a little bit overloaded, its really hard to understand what is going on. I would recommend to stick to KISS principle here. You dont need array assignment. I guess you can try arranging the image margin from css, not js (+ keep in mind that there is an issue with image stretching); and your code would be `const {images: {fixed_width: {url}}} = data[key];`

Reply:
and related to KISS, no need in array assignment for such dummy operation as `margin = '0.5rem 0.5rem';`. You can always just leave it